templates/template_vars.yaml

```diff
@@ -0,0 +1,43 @@
+README:
+  filename: README.md
+  template: genai-perf-templates/README_template
+  release: 24.08
```

Comment:
Great work!

Is there a way to centralize the variables here? It seems like this moves us from needing to find the variables to update to being able to update them in one file then run the script. The next step for me would be to just having one centralized list of variables (maybe in this file?) to then know we need to update each one once. We'd go to the top of the and update release version, CLI version, etc. once and be done.

Reply:
This was a conscientious decision on my part to provide flexibility for every file while still centralizing all variables.
I can see going forward making a central list and then only overwrite if it is specified for a file. However, I did not want to over engineer this right now when ops is likely to take this completely out of this repo.
I wanted a short term solution that had room to evolve for later.

For now, search and replace in this one file is still very straight forward.

Reply:
Yeah I had the same suggestion. I had your branch pulled and was working on it when the suggestion came through. Seems like it's not as straight-forward as I'd hoped, though I'm no YML expert. I think we can extend this in the future.

Reply:
Updated the script to centralize the variables to  a single general set.
Now each file may specialize a variable which will take precedence over the general variable if both are defined.
Thank you for the suggestion, @dyastremsky  and @fpetrini15!